### PR TITLE
如果滚动条在body上下拉向上向下位置计算有误

### PR DESCRIPTION
**Pull request by machuanjia**
uiSelect：directive
var _scrollTop = $document[0].documentElement.scrollTop;
if(_scrollTop === 0){
  _scrollTop = angular.element("body")[0].scrollTop;
}
// Determine if the direction of the dropdown needs to be changed.
if (offset.top + offset.height + offsetDropdown.height > _scrollTop + $document[0].documentElement.clientHeight) {

### CODE REVIEW

**Comment by cdjackson** [comment]
probable duplicate of #1047

**Comment by cdjackson** [comment]
probable duplicate of #989

**Comment by cdjackson** [comment]
#1016 #990 #836 #977 #148

File: src/uiSelectDirective.js [hunk 1]
```diff
@@ -273,7 +273,11 @@ uis.directive('uiSelect',
               var offsetDropdown = uisOffset(dropdown);
 
               // Determine if the direction of the dropdown needs to be changed.
-              if (offset.top + offset.height + offsetDropdown.height > $document[0].documentElement.scrollTop + $document[0].documentElement.clientHeight) {
+              var _scrollTop = $document[0].documentElement.scrollTop;
+              if(_scrollTop === 0){
+                _scrollTop = angular.element("body")[0].scrollTop;
```
Comment (cdjackson) [inline src/uiSelectDirective.js]: Is this supported by jqLite or are you requiring jQuery?

**Comment by dimirc** [comment]
Should be solved with https://github.com/angular-ui/ui-select/pull/1212

**Pull request closed without merging**